src/test/components/__snapshots__/MenuButtons.test.js.snap

```diff
@@ -683,7 +683,7 @@ exports[`app/MenuButtons <MetaInfoPanel> matches the snapshot 1`] = `
       <summary
         class="arrowPanelSubTitle"
       >
-        Profiler Overhead
+        ⁨Profiler⁩ Overhead
```

Comment:
I have the feeling there are hidden unicode characters (directionality) here because of the terms, but they don't show up on GitHub.

Reply:
Yeah, I'm pretty sure this exactly this